README.md

```diff
@@ -1 +1,98 @@
 # Frames Adapter
+
+Frames Adapter is a small typescript library designed to validate the payload from a signed frames POST call for DSCVR frames.
```

Comment:
frames-adapter is a small typescript library that can be used as an adapter to different frames frameworks. Currently, frames.js is supported. The library provides a couple of helper methods to validate the trusted message data of a frame action and also to determine if a frame is using the DSCVR protocol.

Reply:
Fixed